quiz/src/components/Category.js

```diff
@@ -55,6 +56,7 @@ function Category() {
             </select>
           </div>
           <button
+            className="submitBtn"
             type="submit"
             onClick={(e) => {
               setShow(!show);
```

Comment:
you should fix the case when there is no request is sent , that means at the first time when the values of `cat` & `diff` are null,  so, you should check if `cat` && `diff` aren't null before go to questions.